Dockerfile

```diff
@@ -1,5 +1,6 @@
-ARG CUDA_TAG=9.2-devel
-FROM nvidia/cuda:$CUDA_TAG
+ARG CUDA_TAG=10.2
+ARG OS_TAG=18.04
+FROM alicevision/popsift:deps-cuda${CUDA_TAG}-ubuntu${OS_TAG}
```

Comment:
To be consistent with AliceVision:
`alicevision/popsift-deps:cuda${CUDA_TAG}-ubuntu${OS_TAG}`
Or should we change on the other side?

Reply:
ah i didn't see that, I was just following Nvidia pattern `nvidia/cuda:${CUDA_TAG}-devel-ubuntu${OS_TAG}`

Reply:
ah no I thought u were referring to the `-` instead of `_`.
I put it like that because I thought on docker hub the image need to have the same name as the repository hosting it.